R/dettl_runner.R

```diff
@@ -51,6 +51,9 @@ run_import <- function(import, run_stages = NULL, dry_run = FALSE) {
     stop(
       "Can only run import for non null data import with class 'DataImport'.")
   }
+  if (!dry_run && !git_is_clean(import$path)) {
+    stop("Can't run import as repository has unstaged changes. Update git or run in dry-run mode.")
```

Comment:
I would suggest adding a `force` argument here too perhaps (with default value `FALSE`) or we might find that things get a bit restrictive (i.e, the condition becomes `!force && !dry_run && !git_is_clean(import$path)`

Reply:
That's a good idea, I've added that.